index.d.ts

```diff
@@ -0,0 +1,61 @@
+declare module "react-native-photo-upload" {
+  import { StyleProp, ViewStyle } from "react-native";
+
+  interface ImagePickerResponse {
+    customButton: string;
+    didCancel: boolean;
+    error: string;
+    data: string;
+    uri: string;
+    origURL?: string;
+    isVertical: boolean;
+    width: number;
+    height: number;
+    fileSize: number;
+    type?: string;
+    fileName?: string;
+    path?: string;
+    latitude?: number;
+    longitude?: number;
+    timestamp?: string;
+  }
+
+  interface ImageResizerResponse {
+    path: string;
+    uri: string;
+    size?: number;
+    name?: string;
+  }
+
+  type PhotoUploadProps = {
```

Comment:
Why not `interface`?

Reply:
whoops, should be also an interface. fixed in the second commit

Reply:
👍